socorro/external/postgresql/postgresqlalchemymanager.py

```diff
@@ -52,9 +63,10 @@ def setup_admin(self):
         if not self.min_ver_check(90300):
             self.session.execute(
                 'CREATE EXTENSION IF NOT EXISTS json_enhancements')
-        self.session.execute('CREATE SCHEMA bixie')
-        self.session.execute(
-            'GRANT ALL ON SCHEMA bixie, public TO breakpad_rw')
+        self.session.execute('CREATE SCHEMA IF NOT EXISTS bixie')
```

Comment:
didn't we remove the bixie stuff elsewhere? Surprised it's still in here could probably drop it at this point.